src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -29,6 +31,75 @@ declare global {
+     */
+    get isToken(): boolean;
+
+    /**
+     * An array of ActiveEffect instances which are present on the Actor which have a limited duration.
+     */
+    get temporaryEffects(): ActiveEffect[];
+
+    /**
+     * Return a reference to the TokenDocument which owns this Actor as a synthetic override
+     */
+    get token(): ConfiguredDocumentClassOrDefault<typeof foundry.documents.BaseToken> | null;
+    /**
+     * A convenience reference to the item type (data.type) of this Actor
+     */
+    get type(): this['data']['type'];
+    /**
+     * Apply any transformations to the Actor data which are caused by ActiveEffects.
+     */
+    applyActiveEffects(): void;
+
+    /**
+     * Retrieve an Array of active tokens which represent this Actor in the current canvas Scene.
+     * If the canvas is not currently active, or there are no linked actors, the returned Array will be empty.
+     * If the Actor is a synthetic token actor, only the exact Token which it represents will be returned.
+     *
+     * @param linked - Limit results to Tokens which are linked to the Actor. Otherwise return all Tokens even those which are not linked.
+     * @param document - Return the Document instance rather than the PlaceableObject
+     * @returnS An array of Token instances in the current Scene which reference this Actor.
+     */
+    getActiveTokens(linked?: boolean, document?: boolean): Token[];
+    /**
+     * Prepare a data object which defines the data schema used by dice roll commands against this Actor
+     */
+    getRollData(): this['data']['data'];
+    /**
+     * Create a new TokenData object which can be used to create a Token representation of the Actor.
+     * @param data - Additional data, such as x, y, rotation, etc. for the created token data
+     * @returns The created TokenData instance
+     */
+    getTokenData(data: object): Promise<object>;
+    /**
+     * Get an Array of Token images which could represent this Actor
+     */
+    getTokenImages(): Promise<string[]>;
+
+    /**
+     * Roll initiative for all Combatants in the currently active Combat encounter which are associated with this Actor.
+     * If viewing a full Actor entity, all Tokens which map to that actor will be targeted for initiative rolls.
+     * If viewing a synthetic Token actor, only that particular Token will be targeted for an initiative roll.
+     *
+     * @param options - Configuration for how initiative for this Actor is rolled.
+     * @param options['createCombatants'] - Create new Combatant entries for Tokens associated with this actor.
+     * @param {boolean} [options.rerollInitiative=false]    Re-roll the initiative for this Actor if it has already been rolled.
+     * @param {object} [options.initiativeOptions={}]       Additional options passed to the Combat#rollInitiative method.
+     * @returns         A promise which resolves to the Combat entity once rolls are complete.
+     */
+    rollInitiative(options?: {
```

Comment:
The eslint warnings should be fixed. Further the defaults should be documented. And finally the `initiativeOptions` should be typed if possible or a `TODO` comment added.